### ISSUE TITLE
How to use torch.nn.functional.normalize in torch2trt

### ISSUE THREAD

**Issue by reborm** [NVIDIA-AI-IOT/torch2trt#60]
Has anyone translated the normalize function of pytorch into the torch2trt?

**Comment by jaybdub** [comment]
Hi reborm,

Thanks for reaching out!

I think this should be possible using standard TensorRT operations, but I need to look into it a bit further.

I'll keep you updated on this thread.

Best,
John

**Comment by jaybdub** [comment]
Hi reborm,

I've implemented conversion of ``torch.nn.functional.normalize`` using standard TensorRT operations.

For your reference, and to help guide others trying to implement converters, the implementation is here

https://github.com/NVIDIA-AI-IOT/torch2trt/blob/493515c1df4d06e2c28f34aac988758c224dcd4f/torch2trt/converters/normalize.py#L33

It has been merged into master, you can test it by reinstalling torch2trt and calling

```bash
python3 -m torch2trt.test --name=normalize
```

Best,
John

**Comment by reborm** [comment]
Wow, Thank you so much!!!   I'm shocked you reply so quickly
